go.mod

```diff
@@ -13,3 +13,62 @@ require (
 	k8s.io/kubectl v0.22.2
 	sigs.k8s.io/controller-runtime v0.10.2
 )
+
+require (
```

Comment:
This is a result of running "go mod tidy" with go1.1.7.1

Reply:
ach, forget that ^